packages/SwingSet/test/promise-watcher/test-promise-watcher.js

```diff
@@ -67,7 +68,7 @@ async function testPromiseWatcher(t) {
     'p2-dk.rej rejected "err2" version v1 via VDO (rej)',
   ];
   t.deepEqual(c.dump().log, beforeReference);
-  if (os.platform() === 'darwin') {
+  if (os.platform() === 'darwin' && workerType === 'local') {
```

Comment:
huh, I'd like to resolve this better, if we can figure out the underlying issue. Switching on "darwin" feels pretty wrong to me.